engine/servermaster/jobmanager.go

```diff
@@ -177,6 +183,12 @@ func (jm *JobManagerImpl) GetJob(ctx context.Context, req *pb.GetJobRequest) (*p
 		return nil, err
 	}
 
+	// if job status is running, forward the request to jobmaster openapi
+	if masterMeta.StatusCode == frameModel.MasterStatusInit {
```

Comment:
Can add an API to status code, such as `CanQueryJobDetail` and reuse it

Reply:
Fix it.